package.json

```diff
@@ -238,37 +238,37 @@
     "compile": "webpack --mode production",
     "watch": "webpack --mode development --watch --info-verbosity verbose",
     "lint": "tslint -c tslint.json 'src/**/*.ts' 'tests/**/*.ts'",
-    "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "jest",
     "watch-test": "yarn test -- --watch",
     "prettier": "prettier",
     "prettier-write": "yarn prettier --parser typescript --no-semi --single-quote --trailing-comma es5 --write --print-width 120",
     "prettier-project": "yarn prettier-write -- '?(__mocks__|src|tests)/**/*.ts'"
   },
-  "devDependencies": {
-    "@types/jest": "^24.0.11",
-    "@types/node": "^8.0.31",
-    "coveralls": "^3.0.0",
-    "danger": "^2",
-    "glob": "^7.1.1",
-    "husky": "^0.14.3",
```

Comment:
We don't need `husky` anymore?

Reply:
I can't find any reference of it in our code base... do you recall why it was added?

Reply:
It was added here:
https://github.com/jest-community/vscode-jest/commit/99154c39cf458cac8b6356a1b6f401c42a1ff34a#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R114

I'm thinking it might come along with `lint-staged`. From [their docs](https://github.com/okonet/lint-staged#installation-and-setup):

> The fastest way to start using lint-staged is to run following command in your terminal:
> 
> ```
> npx mrm lint-staged
> ```
> **It will install and configure [husky](https://github.com/typicode/husky)** and lint-staged depending on code quality tools from package.json dependencies so please make sure you install (npm install --save-dev) and configure all code quality tools like Prettier, ESlint prior that.

Reply:
looks like git command used its own husky and able to execute the "precommit" in package.json without the local husky installation... maybe something has changed after initial husky PR, almost 3 years ago? Anyway, given everything is working as expected, I think it's probably safe to remove husky.

However I did discover our lint-staged config and prettier commands are outdated and not ideal, so took the chance to clean them up as well.